setup.py

```diff
@@ -2,6 +2,13 @@
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
+long_description += """
+
+****************
+
+"""
+with open("CHANGELOG.md", "r") as fh:
+    long_description += fh.read()
```

Comment:
I prefer link to the online changelog instead of including it here. I don't see much value in duplicating it. Besides, it is soon going to be longer than the actual readme which is not good either.

Reply:
OK, thanks - I will drop it for now and add/test the change log later.